f8a_worker/workers/bookkeeper.py

```diff
@@ -71,11 +71,14 @@ def execute(self, arguments):
         self._strict_assert(arguments.get('data'))
 
         aggregated = ''
-        if arguments['data'].get('api_name') == 'stack_analyses' and \
+        if arguments.get('api_name') == 'stack_analyses' and \
```

Comment:
Why have you changed the code for stack analyses. Does it mean that is not working currently in prod.

Reply:
@prashbnair   I am not sure whats happening in prod.But that was like a syntax error that I saw in the worker. "string does not have get() attribute".

Reply:
resolved.